packages/styleguide/src/components/Center/index.tsx

```diff
@@ -113,9 +113,9 @@ const centerStyles = {
 
 type CenterProps = {
   children: React.ReactNode
-  attributes?: React.HTMLAttributes<HTMLDivElement>
+  attributes?: React.ComponentPropsWithoutRef<'div'>
   breakout?: boolean
-} & React.HTMLAttributes<HTMLDivElement>
+} & React.ComponentProps<'div'>
```

Comment:
I think `ComponentPropsWithoutRef` should be used everywhere except in cases where we _do_ forward the ref?

Reply:
```suggestion
} & React.ComponentPropsWithoutRef<'div'>
```